test/cuda_sims.jl

```diff
@@ -274,7 +274,7 @@ end
          a = 0.94f0,
          ν = 182.5f0)
 
-  tₑ = 3.0f0
+  tₑ = 1.0f0
```

Comment:
Same as before, is this enough to see at least a single ridge form and stabilize?